sdk/avs/Microsoft.Azure.Management.Avs/src/Microsoft.Azure.Management.Avs.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Avs</PackageId>
     <Description>Provides developers with libraries for Azure VMware Solution.</Description>
-    <Version>1.0.0-preview.1</Version>
```

Comment:
This is meant to be a non-preview package?  What is the api-version this is based on (should be non-preview).

Reply:
Yes, it is meant to be non-preview. AVS service went GA on 9/28 with this spec: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/vmware.json